python/acl_anthology/collections/paper.py

```diff
@@ -80,7 +86,7 @@ class Paper:
     bibkey: str = field()
     title: MarkupText = field()
 
-    attachments: dict[str, AttachmentReference] = field(factory=dict, repr=False)
+    attachments: list[tuple[str, AttachmentReference]] = field(factory=list, repr=False)
     authors: list[NameSpecification] = Factory(list)
```

Comment:
Because inferring author identities (and definitely providing meaningful attributes for these author objects) requires loading every single XML file first.

Reply:
I wonder where a good place would be to discuss these things; issues or comments in larger PRs seem suboptimal for that. The "discussion" tab maybe? (I've never used that)